### DISCUSSION TITLE
유니티에서 play on Zepeto 오류

### DISCUSSION THREAD

**Discussion by uniconnect147** [naverz/zepeto-studio-kor#1037] [5. Q&A]
qr코드는 뜨지만 보라색화면만 나오고 오래두면 오류메세지가 뜹니다.
예전에 올라온글에서 네트워크를 같게해주라고 하시는데 어떤 의미인가요??
저는 현재 데스크탑에 이더넷선을 연결해 사용중이고 핸드폰은 그 이더넷선이 연결된 공유기 와이파이를 쓰고 있습니다~

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 이더넷선이 연결된 공유기에서 랜선을 꽂아서 PC를 연결하고 모바일 기기가 와이파이를 사용하면 동일한 네트워크 환경으로 인식할 것 같습니다.

**Reply by uniconnect147** [reply]
네! 현재 그상태인데 계속 안돼요 ㅠㅜ

**Reply by Zepeto-TechEvangelist** [reply]
방화벽 설정도 보시면 좋을 것 같습니다. 아래 이슈 참고 부탁 드립니다. https://github.com/naverz/zepeto-studio-kor/discussions/591
